pkg/vsphere/disk/disk_manager.go

```diff
@@ -508,7 +508,14 @@ func (m *Manager) AttachAndMount(op trace.Operation, datastoreURI *object.Datast
 		return "", err
 	}
 
-	return d.Mount(op, nil)
+	// don't update access time - that would cause the diff operation to mutate the filesystem
+	opts := []string{"noatime", ""}
```

Comment:
I've no idea why we have the extra "" in there - I don't think it's needed.

I'm also not sure what happens here when combined with a copyTo operation that needs a writable mount. The writable mount should have `persistent == true` but don't know whether that's actually possible at the same time as a non-persistent mount. Hopefully we're not using `!persistent` atm ;)